### DISCUSSION TITLE
Draggable components

### DISCUSSION THREAD

**Discussion by auronvila** [mantinedev/mantine#7265] [Feature requests]
I wonder if mantine is having a plan in the near future to add drag ability to it's components.

## Example Draggable table
it would be nice to have a draggable table option where we could rearrange the rows inside of the table.

## Example Draggable grid 
in a grid there can be customizable cards where we could drag and drop them, you can think of it like a kanban board but with additional features.

**Comment by rtivital** [comment]
It is outside the scope of UI libraries, for these features, use dedicated dnd libraries line dndkit.

**Reply by auronvila** [reply]
Thank you for clarification.

Greetings
